src/test/java/com/patreon/PatreonAPITest.java

```diff
@@ -0,0 +1,48 @@
+package com.patreon;
+
+import com.github.jasminb.jsonapi.JSONAPIDocument;
+import com.patreon.PatreonAPI;
+import com.patreon.resources.Campaign;
+import com.patreon.resources.Pledge;
+import com.patreon.resources.User;
+import junit.framework.Assert;
+import junit.framework.TestCase;
+
+import java.util.List;
+
+public class PatreonAPITest extends TestCase {
+    private PatreonAPI api = new PatreonAPI("a token");
+
+    public void testFetchCampaigns() throws Exception {
+        JSONAPIDocument<List<Campaign>> campaignResponse = api.fetchCampaigns();
+        Assert.assertEquals(1, campaignResponse.get().size());
+        Campaign campaign = campaignResponse.get().get(0);
+        Assert.assertEquals("70261", campaign.getId());
```

Comment:
pull expected campaign data into constant?

Reply:
:+1: there's more extensive stuff to do rewriting all tests -- right now they require you to add a real access_token, rather than mocking network